packages/sdk/src/sdk/sdk.deposit.ts

```diff
@@ -164,13 +160,15 @@ export async function getDepositData<
       ...config.source,
     },
     sourceBalance,
+    sourceAssetDecimals:
+      config.source.assetsDecimals?.[asset.originSymbol] || decimals,
     sourceFeeBalance: !isUndefined(sourceFeeBalance)
       ? { ...meta, balance: sourceFeeBalance }
       : undefined,
     sourceMinBalance,
     getFee: async (amount = sourceBalance): Promise<bigint> => {
       const extrinsic = createExtrinsic(amount);
-      const info = await extrinsic.paymentInfo(sourceAccount);
+      const info = await extrinsic.paymentInfo(sourceAccount, { nonce: -1 });
```

Comment:
Why is the nonce required?

Reply:
Without this, Equilibrium calls are failing. This is a solution that they provided and looks like it doesn't harm for other calls.

Reply:
I don't like the fact that we are applying it to all the calls, but it does not seem to affect, at least in functionality, as Elmar suggests.